src/main/resources/akka.conf

```diff
@@ -29,6 +29,9 @@ akka {
 
       # Log implicitly registered classes. Useful, if you want to know all classes which are serialized.
       implicit-registration-logging = true
+
+      # enable compression
+      post-serialization-transformations="lz4"
```

Comment:
formatting + add an env variable "KRYO_COMPRESSION" to be able to change the conf